src/posit/connect/bundles.py

```diff
@@ -1,17 +1,14 @@
 from __future__ import annotations
 
-from typing import List
+from typing import BinaryIO, List
 
 from requests.sessions import Session as Session
```

Comment:
`import Session as Session`?

Reply:
Ha, that was vscode auto import. I'll fix.